jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -371,15 +371,16 @@ public void cancel(InterpreterContext context) {
   }
 
   public String getPropertyKey(String cmd) {
-    int firstLineIndex = cmd.indexOf("\n");
-    if (-1 == firstLineIndex) {
-      firstLineIndex = cmd.length();
-    }
-    int configStartIndex = cmd.indexOf("(");
-    int configLastIndex = cmd.indexOf(")");
-    if (configStartIndex != -1 && configLastIndex != -1
-        && configLastIndex < firstLineIndex && configLastIndex < firstLineIndex) {
-      return cmd.substring(configStartIndex + 1, configLastIndex);
+    boolean firstLineIndex = cmd.startsWith("(");
+
+    if (firstLineIndex) {
```

Comment:
This presupposes that the end-user will be using the first line as a declaration of the interpreter:
`%jdbc (redshift)\n`
What about in cases where the user has selected a default interpreter for each paragraph and doesn't need to specify the `%<interpreter>` prefix? When a user does that then any query with a function will error.